executors/node/collator.js

```diff
@@ -30,11 +69,10 @@ module.exports = {
         testCollOptions['sensitivity'] = 'accent';
       } else
       if (strength == 'tertiary') {
-        testCollOptions['sensitivity'] = 'case';
+        testCollOptions['sensitivity'] = 'variant';
```

Comment:
Hmm, how many cases does this fix?

Reply:
I'm not sure but will try to figure this out.